doc/ref_model/chapters/chapter08-annex.md

```diff
@@ -6,232 +6,133 @@
-<ul>
-<li>nova-api-metadaa</li>
-<li>nova-conductor</li>
-<li>nova-scheduler</li>
-<li>nova-placement-api</li>
-<li>nova-console-auth</li>
-<li>nova-novnc-proxy</li>
-<li>nova-rabbitmq</li>
-<li>glance-api</li>
-<li>glance-registry</li>
-<li>glance-rabbitmq</li>
-<li>heat-api</li>
-<li>heat-cfn</li>
-<li>heat-engine</li>
-<li>heat-rabbitmq</li>
-<li>keystone-api</li>
-<li>keystone-rabbitmq</li>
-<li>keystone-rabbitmq</li>
-</ul>
-</li>
-<li>Validate maridb cluster is insync
-<ul>
-<li>Studown mariadb and upon restart ensure its sync up with masterdb.</li>
-<li>Maria DB is single point of failure</li>
-</ul>
-</li>
-<li>Validate ceph
-<ul>
-<li>Restart ceph-osd</li>
-<li>Document VNF impact while ceph is unavailable and once ceph service is restoredbeing restored.</li>
-</ul>
-</li>
-</ul>
-</li>
-<ul>
-<li><span style="text-decoration: underline;"><strong>Security - see Ch 7 for complete list</strong></span>
-<ul>
-<li>Validation above is performed using both RBAC Roles and User group policies, both of Admin and User Roles.
-</ul>
-<li>**SUMMARY:** Validate User Role to ensure it allow user to perform all designated task and prohibits user performing any unassigned task.</li>
-<li>Validate Security Policy/Rules are enforced</li>
-</ul>
-</li>
-</ul>
-</ul>
+**FuncTest**
+
+FuncTest test suites must be run as part of NFVi validation against all OpenStack based reference implementations. At a minimum, FuncTest executions will include execution of the functest-smoke and functional-benchmarking test suites.
+
+Additional Resources for FuncTest:
+- [Project Description](https://wiki.opnfv.org/display/functest/Opnfv+Functional+Testing#OpnfvFunctionalTesting-Testcases)
+- [User Guide](https://opnfv-functest.readthedocs.io/en/stable-hunter/testing/user/userguide/index.html)
+- [Test Suite Overview](https://opnfv-functest.readthedocs.io/en/stable-hunter/testing/user/userguide/test_overview.html)
+
+Example, or Reference, Functest Status Reporting Artifacts:
+- [Rally Verification](http://artifacts.opnfv.org/functest/functest-opnfv-functest-smoke-iruya-tempest_full-run-259/results/tempest_full/tempest-report.html)
+- [Rally Tasks](http://artifacts.opnfv.org/functest/functest-opnfv-functest-benchmarking-iruya-rally_full-run-169/results/rally_full/rally_full.html)
+
+**Baremetal**
+**SUMMARY:** Control and compute node hardware, bios, firmware, network interfaces, and base operating system configuration must be validated.
+- Interface - Validate nic status for all member in bond1 group
+- Interface - MTU speed for bond1 interface
+- Grub - SR-IOV is enabled
+- Numa - Validate numa configuration on each node
+- Numa - Validate total memory available is distributed equally across numa boundaries
+- VFs - Validate VFs are availablebeing created on each PCI SYS Interface.
+- Modules - Validate the following modules are loaded
+  - bonding
+  - 8021q
+  - i40e
```

Comment:
Isn't this a specific NIC driver?

Reply:
> I suggest @kagreenwell to do as much cleanup as possible in here, get it approved and merged. Then we will have another issue to move this content to the relevant WS (as per after re-structuring).

Rabi-
We are aligned. Mike and I had the same conversation. This cleanup is being done with the expectation that the results of it will be incorporated into the OpenStack specific RI documents when the rewrite is done.

Reply:
Made this significantly more generic.